### ISSUE TITLE
Promote 3.4 to beta

### ISSUE THREAD

**Issue by qmfrederik** [dotnet/Nerdbank.GitVersioning#570]
The 3.4 branch contains the managed git implementation and has been in alpha for the past couple of months.

I don't think we've gotten any new bugs w.r.t. the managed Git implementation, so perhaps it makes sense to promote 3.4 to beta?

**Comment by AArnott** [comment]
Closed with 3c410a85e6f075a9081ac11ec06dac271ed73b8a

Before going stable, I'd like to review active bugs and learn why two docker images recently had to be removed from our testing pipeline to see if we need to fix something to bring them back.
If you have ideas for any of these it may help expedite release.

**Comment by qmfrederik** [comment]
Thanks, @AArnott .

Regarding active bugs, I skimmed the bug list before opening the issue and didn't find any which immediately caught my attention as related to the managed Git implementation.

Regarding the containers, I had a quick look and it seems you've removed Ubuntu Xenial and Arch Linux, right?

The failures seem unrelated at first sight - the Arch Linux build is failing because `dotnet --info` fails to bind to the correct GLIBC version. It's odd because it looks like `/usr/lib/libunwind-x86_64.so.8` is expecting a specific version of GLIBC in `/usr/lib/libc.so.6`. I'd guess both libunwind and libc are provided by the OS itself; so it's odd there's an inconsistency there?

The Ubuntu Xenial issue seems related to the DevOps agent itself (and perhaps transient?)

[Arch Linux](https://dev.azure.com/andrewarnott/OSS/_build/results?buildId=4388&view=logs&jobId=f6f3752a-4e86-5a23-8978-82c901f2777b&j=5d455352-d41f-5371-7ff5-eff6b6dbd088&t=f3f2660d-3aa8-5f7f-6138-2508c609ad54):

```
Starting: Show dotnet SDK info
==============================================================================
Task         : Command line
Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
Version      : 2.182.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line
==============================================================================
Generating script.
Script contents:
dotnet --info
========================== Starting Command Output ===========================
/usr/sbin/bash --noprofile --norc /__w/_temp/4ba7cd58-9ff4-4315-bc3c-0da18cfcaaba.sh
Failed to load , error: /usr/lib/libc.so.6: version `GLIBC_2.33' not found (required by /usr/lib/libunwind-x86_64.so.8)
Failed to bind to CoreCLR at '/usr/share/dotnet/shared/Microsoft.NETCore.App/3.1.8/'
Failed to create CoreCLR, HRESULT: 0x80008088

Host (useful for support):
  Version: 3.1.8
  Commit:  05a0c8f6b1

.NET Core SDKs installed:
  3.1.108 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.NETCore.App 3.1.8 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
##[error]Bash exited with code '137'.
Finishing: Show dotnet SDK info
```

[Ubuntu Xenial](https://dev.azure.com/andrewarnott/OSS/_build/results?buildId=4467&view=logs&j=6ae40e97-b232-5a7d-50d6-16023e515542&t=cedc41bd-5c25-5ee8-1fa8-b044580ce2c6)

```
Starting: Configure container
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.182.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
OCI runtime exec failed: exec failed: container_linux.go:367: starting container process caused: chdir to cwd ("/root") set in config.json failed: permission denied: unknown
##[error]Exit code 126 returned from process: file name '/usr/bin/docker', arguments 'exec -i -u 1001 c9c246215800f23ec21c5484bf3e4da9463f08d824313a4c7893ec5ca9824be5 /__a/externals/node/bin/node /__w/_temp/containerHandlerInvoker.js'.
Finishing: Configure container
```
